### ISSUE TITLE
Address Model setName

### ISSUE THREAD

**Issue by xinhus** [konduto/php-sdk#9]
In Address Model the function setName are setting the wrong attribute.

```
    public function setName($value) {
        $this->set("zip", $value);
    }
```
Instead 
```
    public function setName($value) {
        $this->set("name", $value);
    }
```

**Comment by adri0** [comment]
Thanks @xinhus!

**Comment by adri0** [comment]
Fixed in release [v2.0.1](https://github.com/konduto/php-sdk/releases/tag/v2.0.1)
